src/main/java/dev/j3fftw/litexpansion/LiteXpansion.java

```diff
@@ -99,11 +99,15 @@ private void changeSfValues() {
             Reflections.setField(energizedPanel, "nightEnergy", 32);
         }
 
-        // InfinityExpansion - Halved all values and made infinite panel much less
-        Reflections.setField(SlimefunItem.getByID("ADVANCED_PANEL"), "generation", 75);
-        Reflections.setField(SlimefunItem.getByID("CELESTIAL_PANEL"), "generation", 250);
-        Reflections.setField(SlimefunItem.getByID("VOID_PANEL"), "generation", 1200);
-        Reflections.setField(SlimefunItem.getByID("INFINITE_PANEL"), "generation", 20_000);
+        if (getConfig().getBoolean("options.nerf-other-addons")) {
```

Comment:
make this config option work for all nerfs. also renamek it to options.weakling and at some comments above it in config. "This setting makes LiteXpansions way easier. This will be handy for people without the will of grinding" something like this. might change it later